### PR TITLE
added support for OS X

### PR DESCRIPTION
**Pull request by digitalmoksha**
Made changes to allow usage and testing for OS X apps.  To test, just do `rake spec platform=osx`.  The default platform is iOS.

### CODE REVIEW

File: spec/colorizer.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
       {rubymotion: "rocks!"},
       Time.now,
       Object.new,
-      UIView.new,
+      NSSet.new,
```
Comment (markrickert) [inline spec/colorizer.rb]: At first I was wondering why you took out UIView in the test and then realized it doesn't exist on Mac :)

**Comment by markrickert** [comment]
Dude... this is GREAT! Thank you so much!

**Comment by digitalmoksha** [comment]
Yeah, no problem.  I'm now using it in one of my OS X projects!

Hope you don't mind, I have another PR coming...

File: spec/colorizer.rb (hunk 1 above)
Comment (digitalmoksha) [inline spec/colorizer.rb]: I took the easy way -- It didn't seem worth creating different platform files...

**Pull request merged**